### DISCUSSION TITLE
Where do I find your changelogs for different releases?

### DISCUSSION THREAD

**Discussion by Tejeev** [NicolasPetton/Projectify#7] [Q&A] [answered]
Sorry, I'm just used to seeing them like this https://github.com/rancher/rancher/releases/tag/v2.4.11 so may not be looking in the right places.

**Comment by NicolasPetton** [comment, marked as answer]
I haven't put the changelogs on GitHub tags, but you can find them https://nicolas.petton.fr/tw/projectify.html#ChangeLog.
